### PR TITLE
Fixed a crash in action bar on iOS 9.3

### PR DESCRIPTION
**Pull request by tzraikov**
https://github.com/NativeScript/nativescript-angular/issues/121

### CODE REVIEW

File: tns-core-modules/ui/action-bar/action-bar.ios.ts [hunk 1]
```diff
@@ -112,31 +112,23 @@ export class ActionBar extends common.ActionBar {
...
 
     private populateMenuItems(navigationItem: UINavigationItem) {
-        var items = this.actionItems.getVisibleItems();
-        var leftBarItems = [];
-        var rightBarItems = [];
-
-        for (var i = 0; i < items.length; i++) {
-            var barButtonItem = this.createBarButtonItem(items[i]);
+        let items = this.actionItems.getVisibleItems();
+        let leftBarItems = NSMutableArray.alloc().init();
```
Comment (hshristov) [inline tns-core-modules/ui/action-bar/action-bar.ios.ts]: Use of JS array is recommended as runtimes will marshal it in an optimized way;

File: tns-core-modules/ui/action-bar/action-bar.ios.ts [hunk 2]
```diff
@@ -112,31 +112,23 @@ export class ActionBar extends common.ActionBar {
...
-
-        // Right items should be added in reverse because they are added from right to left
-        var rightArray: NSMutableArray = rightBarItems.length > 0 ? NSMutableArray.new() : null;
-        rightBarItems.reverse();
-        rightBarItems.forEach((barItem, i, a) => rightArray.addObject(barItem));
-
-        navigationItem.leftItemsSupplementBackButton = true;
-        navigationItem.setLeftBarButtonItemsAnimated(leftArray, true);
-        navigationItem.setRightBarButtonItemsAnimated(rightArray, true);
+        navigationItem.setLeftBarButtonItemsAnimated(leftBarItems, false);
```
Comment (hshristov) [inline tns-core-modules/ui/action-bar/action-bar.ios.ts]: Here you can pass directly JS Array.

**Comment by hshristov** [comment]
👍

**Comment by PanayotCankov** [comment]
:+1:

**Pull request merged**